### PR TITLE
Tasks for processing and storing ZkLend protocol events

### PR DESCRIPTION
**Pull request by Benjtalkshow**
#291  closes

### CODE REVIEW

File: apps/data_handler/celery_app/event_tasks.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    except Exception as exc:
+        execution_time = (datetime.utcnow() - start_time).total_seconds()
+        logger.error(
+            f"Error processing ZkLend events after {execution_time:.2f}s: {exc}",
+            exc_info=True
+        )
+        raise
+
+
+if __name__ == "__main__":
```
Comment (djeck1432) [inline apps/data_handler/celery_app/event_tasks.py:46]: remove this one, it's not needed here

**Review by djeck1432 — CHANGES_REQUESTED** [review]
add executing task here
https://github.com/CarmineOptions/derisk-research/blob/master/apps/data_handler/celery_app/celery_conf.py

**Comment by Benjtalkshow** [comment]
> add executing task here https://github.com/CarmineOptions/derisk-research/blob/master/apps/data_handler/celery_app/celery_conf.py

yea. still working on it

File: apps/data_handler/celery_app/event_tasks.py (hunk 1 above)
Comment (Benjtalkshow) [reply apps/data_handler/celery_app/event_tasks.py:46]: Alright. Thanks

**Comment by djeck1432** [comment]
@Benjtalkshow Any updates?

**Comment by Benjtalkshow** [comment]
> @Benjtalkshow Any updates?

Yes, i am trying to run `pylint`

**Comment by Benjtalkshow** [comment]
> @Benjtalkshow Any updates?

Yes, i am trying to run `pylint`

**Comment by Benjtalkshow** [comment]
I have updated my PR 
And made more changes according to this:
image

**Review by djeck1432 — APPROVED** [review]
Looks good

**Pull request merged**